guides/source/form_helpers.md

```diff
@@ -319,29 +319,29 @@ The Article model is directly available to users of the application, so - follow
 resources :articles
 ```
 
-TIP: Declaring a resource has a number of side effects. See [Rails Routing From the Outside In](routing.html#resource-routing-the-rails-default) for more information on setting up and using resources.
+TIP: Declaring a resource has a number of side effects. See [Rails Routing from the Outside In](routing.html#resource-routing-the-rails-default) guide for more information on setting up and using resources.
 
 When dealing with RESTful resources, calls to `form_for` can get significantly easier if you rely on **record identification**. In short, you can just pass the model instance and have Rails figure out model name and the rest:
 
 ```ruby
 ## Creating a new article
 # long-style:
 form_for(@article, url: articles_path)
-# same thing, short-style (record identification gets used):
+# short-style:
 form_for(@article)
 
 ## Editing an existing article
 # long-style:
-form_for(@article, url: article_path(@article), html: {method: "patch"})
+form_for(@article, url: article_path(@article), method: "patch")
 # short-style:
 form_for(@article)
 ```
 
-Notice how the short-style `form_for` invocation is conveniently the same, regardless of the record being new or existing. Record identification is smart enough to figure out if the record is new by asking `record.new_record?`. It also selects the correct path to submit to and the name based on the class of the object.
```

Comment:
This is not internal and it is not useless. Where we would explain this behavior to the users?

Reply:
Agree. I just returned this back.